.github/workflows/frontend-ci.yml

```
@@ -113,6 +113,13 @@ jobs:
       - name: Create any cache tables
         run: python manage.py createcachetable
 
+        # Populate the database with a dandiset for FileBrowser tests
+      - name: Create a superuser
+        run: python manage.py createsuperuser --noinput --email admin@admin.com
```

Since admin.com is a real domain, I'd change this to example.com

```suggestion
        run: python manage.py createsuperuser --noinput --email admin@example.com
```

---

fixed in 1258b94bb37a266cc6262f55b302201715c28300